payjoin-cli/src/app.rs

```diff
@@ -286,20 +286,21 @@ impl App {
         })?;
         log::trace!("check4");
 
-        let mut payjoin = payjoin.identify_receiver_outputs(|output_script| {
-            if let Ok(address) = bitcoin::Address::from_script(output_script, network) {
-                self.bitcoind
-                    .get_address_info(&address)
-                    .map(|info| info.is_mine.unwrap_or(false))
-                    .map_err(|e| Error::Server(e.into()))
-            } else {
-                Ok(false)
-            }
-        })?;
+        let mut provisional_payjoin: ProvisionalProposal =
```

Comment:
is this explicit type required? Seems redundant since you're using such care with naming.

a nit would be to make check4's result `proposal` or `provisional_payjoin` instead of becoming a payjoin and turning into a provisional_payjoin

Reply:
not really, removed.

created a new issue for check4 change here https://github.com/payjoin/rust-payjoin/issues/96, will handle that in a new pr as i dont wanna add more changes to this pr

Reply:
I'm a bit confused why you chose to make this a new issue, it seems this PR introduces the issue, no?